strpipe/toolkit/test/test_build_vocabulary.py

```diff
@@ -42,29 +21,32 @@ def test_build_vocabulary_from_sentences_with_limit_size():
         [
             ['隼興', '喜歡', '蛋白質'],
             ['蛋白質'],
+            ['gb', '的', '下午茶', '是', '香蕉',
+             '和', '茶葉蛋'],
+            ['香蕉', '是', '澱粉'],
+            ['茶葉蛋', '是', '蛋白質'],
         ],
-        vocab_size=5,
+        vocab_size=4,
     )
     assert set(output.keys()) == set(
-        ['<SOS>', '<EOS>', '<PAD>',
-         '<UNK>', '蛋白質'],
+        ['蛋白質', '是', '茶葉蛋', '香蕉'],
     )
     assert set(output.values()) == set(
-        list(range(5)),
-    )
-
-
-def test_build_vocabulary_from_sentences_with_conflict():
-    output = build_vocabulary_from_sentences(
-        [
-            ['<SOS>', '喜歡', '蛋白質', '<EOS>'],
-            ['蛋白質'],
-        ],
-    )
-    assert set(output.keys()) == set(
-        ['<SOS>', '<EOS>', '<PAD>',
-         '<UNK>', '喜歡', '蛋白質'],
-    )
-    assert set(output.values()) == set(
-        list(range(6)),
-    )
+        list(range(4)),
+    )
+
+
+# def test_build_vocabulary_from_sentences_with_conflict():
+#     output = build_vocabulary_from_sentences(
+#         [
+#             ['<SOS>', '喜歡', '蛋白質', '<EOS>'],
+#             ['蛋白質'],
+#         ],
+#     )
+#     assert set(output.keys()) == set(
+#         ['<SOS>', '<EOS>', '<PAD>',
+#          '<UNK>', '喜歡', '蛋白質'],
+#     )
+#     assert set(output.values()) == set(
+#         list(range(6)),
+#     )
```

Comment:
If it's not necessary, simply delete it.

Reply:
8dd916d